model-optimizer/extensions/front/tf/SSliceComplexRolledFFTPackBlockReplacement.py

```diff
@@ -0,0 +1,112 @@
+# Copyright (C) 2018-2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+
+import logging as log
+
+from extensions.middle.InsertLayoutPropagationTransposes import mark_input_as_in_correct_layout
+from extensions.ops.dft import DFT, IDFT
+from extensions.ops.roll import Roll
+from mo.front.common.partial_infer.utils import int64_array
+from mo.front.common.replacement import FrontReplacementSubgraph
+from mo.front.subgraph_matcher import SubgraphMatch
+from mo.front.tf.graph_utils import create_op_with_const_inputs
+from mo.graph.graph import Graph, Node, rename_nodes
+
+
+class SSliceComplexRolledFFTPackBlockReplacement(FrontReplacementSubgraph):
+    enabled = True
```

Comment:
Add description for the transformation.

Reply:
This transformation was split into three transformation, and comments were added to these transformations."